.github/workflows/release.yaml

```diff
@@ -0,0 +1,67 @@
+# Inspired from https://foojay.io/today/how-to-release-a-java-module-with-jreleaser-to-maven-central-with-github-actions/
+name: Publish a new release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version'
+        required: true
+      nextVersion:
+        description: 'Next version after release (-SNAPSHOT will be added automatically)'
+        required: true
+
```

Comment:
Can you please enable the `repository_dispatch` trigger technique? That way the REST API can be used to start the workflow, given a proper Personal Access Token.